lib/core/src/lib/about/about.component.html

```diff
@@ -1,6 +1,6 @@
 <mat-accordion [togglePosition]="'before'" class="adf-about-panel">
     <ng-container *ngFor="let panel of panels">
-        <mat-expansion-panel>
+        <mat-expansion-panel [attr.data-automation-id]="'expansion-panel-' + (panel.label | lowercase).split(' ').join('-')">
```

Comment:
can you make the IDs more predictable? you are messing with the localised label and also force the browser to constantly recalculate the values. Maybe `AboutPanelDirective` needs to receive the automation id similar as the label value, and you can provide automation ids from the application

Reply:
It makes sense, thx, could I just add the new @Input() to the directive to pass the automation id? Example in new commit